pages/1.12/deploying-jobs/metronome-constraints/index.md

```diff
@@ -0,0 +1,140 @@
+---
+layout: layout.pug
+navigationTitle:  Metronome Placement Constraints
+title: Metronome Placement Constraints
+menuWeight: 3
+excerpt: Understanding Metronome placement constraints
+
+enterprise: false
+---
+
+Metronome placement constraints control where jobs run. Constraints have three parts: a field name, an operator, and a value. The field can be the hostname of the agent node or any attribute of the agent node.
+
+Metronome supports a subset of Marathon operators, and supports all of Marathon's special field names.
+
+# Field Names
+
+## Agent properties as a field name
+
+Constraints with a field name beginning with `@` refer to agent properties.
+
+### Host name
+
+Entering `@hostname` as the field name matches the agent node hostnames. See `LIKE operator`, below, for a usage example.
+
+### Region and zone as field names
+
+Use the `@region` and `@zone` field names to configure [fault domain awareness and capacity extension](/1.12/deploying-services/fault-domain-awareness/). See `IS oeprator`, below, for a usage example
+
+## Attribute as field name
+
+If `@hostname`, `@region`, or `@zone` are not specified as field names, then the field name is interpreted as a Mesos agent node attribute. A Mesos agent node attribute allows you to tag an agent node. See `mesos-slave --help` to learn how to set the attributes. If the specified attribute is not defined on the agent node, most operators will refuse to run tasks on it. In fact, only the `UNLIKE` operator will (and always will) accept this offer for now, while other operators will always refuse it.
```

Comment:
this paragraph is confusing... imagine reading this as a newbie...   missing is the definition of "attribute" from the perspective of mesos.   Is there a link reference we can use?   I believe it would be better to explain that there are predefined attributes for DCOS whcih are `@hostname` etc. 
IMO it is also challenging to use the word `operators` with context... since this is written for "operators" of DCOS.
The use of the phrase "this offer" is super confusing...  what is "this"

I might go with:

Mesos defines the ability define attributes at an agent for the purpose of defining special characteristics of an agent such as it has an SSD.  See `mesos-slave --help` to learn how to set the attributes.  Metronome leverages these attributes to provide affinity (or aversion) of a job to that attribute.  Metronome predefines 3 attributes to provide hybrid cloud support which are: `@hostname`, `@region`, or `@zone`.

Reply:
when using the word "operators"...  such as "most operators will refuse"... I would prefix with "most constraint operators..."

Reply:
I'd favor a link over writing about how a Mesos attribute works here. Good point about conflating terms; unfortunately we already have references to Marathon constraint operators all over the docs. I'll clarify by saying constraint operator.

Reply:
(this text was largely copied from the Marathon docs page, shamefully; I dislike copying things twice but did not want to restructure our docs right now)